app/services/preserved_object_handler.rb

```diff
@@ -164,7 +164,7 @@ def moab_validation_errors
   def create_db_objects(status, validated=false)
     results = []
     pp_default = PreservationPolicy.default_preservation_policy
-    create_results = with_active_record_transaction_and_rescue do
+    transaction_results = with_active_record_transaction_and_rescue do
```

Comment:
So `transaction_results` will only have objects in it, if the transaction failed right? And the objects will be error codes if `OBJECT_DOES_NOT_EXIST` or `DB_UPDATE_FAILED` ?

Reply:
If the above is the case then what if we included the word `errors` or something?

Reply:
I think transaction_results are result codes.  note that they get concatenated to results.